lib/topic_creator.rb

```diff
@@ -109,7 +109,13 @@ def watch_topic(topic)
     end
 
     topic.reload.topic_allowed_groups.each do |topic_allowed_group|
-      topic_allowed_group.group.set_message_default_notification_levels!(topic)
+      group = topic_allowed_group.group
+
+      begin
+        group.set_message_default_notification_levels!(topic)
+      rescue Group::GroupPmUserLimitExceededError => e
+        rollback_with!(topic, :too_large_group, group_name: group.name)
+      end
```

Comment:
This is not the most elegant, but it seemed necessary to follow the same patterns used at other places in topic_creator